packages/kbn-rule-data-utils/src/rule_types/stack_rules.ts

```diff
@@ -8,3 +8,4 @@
 
 export const STACK_ALERTS_FEATURE_ID = 'stackAlerts';
 export const ES_QUERY_ID = '.es-query';
+export const ML_ANOMALY_DETECTION_ID = 'xpack.ml.anomaly_detection_alert';
```

Comment:
```suggestion
export const ML_ANOMALY_DETECTION_RULE_TYPE_ID = 'xpack.ml.anomaly_detection_alert';
```

Reply:
I just use the name that was inside of ML but I will change it, I like it!

Reply:
hi @XavierM, I guess you marked this one as resolved by accident. Could you please rename it to `ML_ANOMALY_DETECTION_RULE_TYPE_ID`? 

I noticed that the Observability const name `OBSERVABILITY_THRESHOLD_RULE_TYPE_ID` ends with `RULE_TYPE_ID`, I reckon we should have the same for ML.

Reply:
I think that's done -> e2a8daec8e4c08e8a127f9d162b5a8c75fb97ea5